roles/unionfs/defaults/main.yml

```diff
@@ -32,6 +32,12 @@ mergerfs_release_lookup_command: |
     | jq -r ".assets[] | select(.name | test(\"{{ ansible_distribution | lower }}-{{ ansible_distribution_release | lower }}_amd64\")) \
     | .browser_download_url"
 
+
+mergerfs_release_lookup_command_fallback: |
```

Comment:
as previously stated we won't be adding ways to run on non-LTS versions of Ubuntu, this is a choice for support reasons.